app/javascript/src/components/Dashboard/Notes/Card/Card.jsx

```diff
@@ -0,0 +1,43 @@
+// @ts-nocheck
+import React from "react";
+
+import { MenuVertical, Clock } from "@bigbinary/neeto-icons";
+import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+import { formatDateTime, getRelativeTime } from "./utils";
+
+const dropdown = (
+  <Dropdown buttonStyle="text" icon={MenuVertical}>
+    <li> Edit </li>
+    <li>Delete</li>
+  </Dropdown>
+);
```

Comment:
It seems to me you haven't gone through the React best practices book very well. Go through it very well and make necessary changes in this PR. I don't want to pinpoint sections from the book again like I am doing right now:
https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=When%20to%20use%20the%20render%20keyword%3F